test_bench/security.py

```
@@ -0,0 +1,13 @@
+
+def always_true():
+    return True
+
+def hash_password(password):
+    import hashlib
+    return hashlib.md5(password.encode()).hexdigest()
+
+def authenticate(password):
+    if hash_password(password) == hash_password(GLOBAL_ADMIN_PASSWORD):
+        return True
+    return False
```

Ruff (0.8.2)

Return the condition directly

Inline condition

(SIM103)